src/PscIde/Command.purs

```diff
@@ -277,9 +277,16 @@ instance decodePursuitCompletion :: DecodeJson PursuitCompletion where
       })
 
 instance decodeImportList :: DecodeJson ImportList where
-  decodeJson json = do
-    imports <- decodeJson json
-    pure (ImportList imports)
+  decodeJson json = decodeObject <|> decodeArray
+    where
+      decodeObject = do
+        o <- decodeJson json
+        moduleName <- o .? "moduleName"
+        imports <- o .? "imports"
+        pure (ImportList { moduleName: head moduleName, imports })
```

Comment:
Oh dear, you're right, I just tried on a file with module `Main.Test`:
```
{"result":{"moduleName":["Main","Test"],"imports": ...
```
I can decode it as either string or array of strings to join with `.` so it will work before and after you fix, and we can remove later?